medusa/providers/torrent/html/animetorrents.py

```diff
@@ -245,9 +245,16 @@ def _get_episode_search_strings(self, episode, add_string=''):
             'Episode': []
         }
 
+        season_scene_names = scene_exceptions.get_scene_exceptions(episode.show.indexerid, season=episode.scene_season)
```

Comment:
I'd like to implement something like this in genericprovider. But will possibly modify (or create new) allPossibleShowNames. As there get_scene_exceptions is already called. Just as well return it with the matching season if available.